frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -147,13 +153,46 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
   const [error, setError] = React.useState('');
   const [inProgress, setProgress] = React.useState(false);
   const [selectedNodesCnt, setSelectedNodesCnt] = React.useState(0);
+  const [hasInvalidNodes, setHasInvalidNodes] = React.useState(false);
+  const [nodesWarningMsg, setNodesWarningMsg] = React.useState('');
 
   let storageClass = '';
 
+  const validateCPU = (node: formattedNodeType) => {
+    return Number(_.get(node, 'cells[3].title')) < MIN_NODE_CPU;
```

Comment:
We shouldn't parse the display value in the table. We need to check the value in the original node object.